services/httpd/handler.go

```diff
@@ -845,14 +867,11 @@ type DeleteBody struct {
 // policies".
 func (h *Handler) serveDeleteV2(w http.ResponseWriter, r *http.Request, user meta.User) {
 	db, rp, err := bucket2dbrp(r.URL.Query().Get("bucket"))
+
 	if err != nil {
 		h.httpError(w, fmt.Sprintf("delete - bucket: %s", err.Error()), http.StatusNotFound)
 		return
 	}
-	if db == "" {
```

Comment:
Why does this go away?

Reply:
`bucket2dbrp` handles the case. This was unreachable code.

Reply:
`bucket2dbrp` handles the case. This was unreachable code.